test/int.jl

```diff
@@ -267,6 +267,25 @@ end
     @test typeof(rand(U(0):U(127)) % T) === T
 end
 
+@testset "signed(<:Unsigned), unsigned(<:Signed) for bitstypes" begin
+    for (S,U) in zip(Base.BitSigned_types, Base.BitUnsigned_types)
+        @test signed(U) === S
+        @test unsigned(S) === U
+    end
+end
+@testset "x::[Un]Signed % [Un]Signed returns x" begin
```

Comment:
Just personal taste, but I would join the three `@testsets` together, as they contain the same loop, so it's a bit of a ceremony overhead.